docker/start.py

```diff
@@ -230,7 +263,14 @@ def indexer_no_projects(logger, uri, config_path, sync_period,
 
     wait_for_tomcat(logger, uri)
 
+    periodic_sync = True
+    if sync_period is None or sync_period == 0:
+        periodic_sync = False
+
     while True:
+        if not periodic_sync:
```

Comment:
Should not this check be at the end of `while` loop? It seems more consistent to me or are we trying to prevent even the 1st invocation?

Reply:
I have not made my mind yet whether zero period should disable the reindex altogether. This preserves current behavior.